lib/cfe_internal_hub.cf

```diff
@@ -202,8 +202,14 @@ bundle agent cfe_internal_database_cleanup_consumer_status (row_count)
 {
 
   vars:
+
+      "status_table_name" -> { "ENT-4331" }
+        string => ifelse( eval( "$(sys.cf_version) < 3.12.1", class, infix ), "__status",
+                          "status"),
```

Comment:
Either a don't understand how `eval` and `ifelse` work or this is wrong. The name of the table is `status` on 3.12.1 and older, `__status` on 3.12.2 and newer.

Reply:
Oh i put it in backwards.
DMANit